frontend/src/actions/transfers/transfer.ts

```diff
@@ -19,7 +19,7 @@ import type { RequestInformationData } from './request-information';
 import { RequestInformation } from './request-information';
 
 const STEPS_DATA = [
-  { identifier: 'sendRequestTransaction', label: 'Please confirm your request on Metamask' },
+  { identifier: 'sendRequestTransaction', label: 'Please confirm the transaction' },
```

Comment:
Okay interesting. I can clarify this again. 👍🏾 
Btw: this got shorten to make it fit without line break in the new UI.

Reply:
Alright. I talked with Olympe about it.  And especially because of having WalletConnect now and the user should be informed to pick up his phone and confirm, she really likes it. She was unsure when this conversation was anyway. 😆 

Thank you for ensuring that we have the right concept in place here!

Reply:
> Btw: this got shorten to make it fit without line break in the new UI.
Oh, I thought it was a proactive action for the coming WC integration :D



> she really likes it. She was unsure when this conversation was anyway. 😆
That's normal, maybe I was imagining this as well! 🤣